.github/workflows/run_notebooks.yml

```diff
@@ -0,0 +1,58 @@
+name: Run notebooks
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 13 * * *'
+
+env:
+  PYTHON_VERSION: "3.11"
```

Comment:
might be worth parameterizing this to make sure they run in 3.9 + 3.12 mostly

Reply:
Will do separately as it's easier to debug workflow dispatch after merging the new workflow to master.

Reply:
Added python version as a parameter in https://github.com/langchain-ai/langchain/pull/26956